packages/components/src/components/popover/popover.tsx

```diff
@@ -64,26 +68,22 @@ export class Popover {
 
   public render(): JSX.Element {
     validateProps(this, propTypes);
-    attachComponentCss(this.host, getComponentCss, this.direction);
+    attachComponentCss(this.host, getComponentCss, this.direction, this.theme);
 
     const PrefixedTagNames = getPrefixedTagNames(this.host);
 
     return (
       <Host onKeydown={this.onKeydown}>
-        <PrefixedTagNames.pButtonPure
+        <button
```

Comment:
Why issn' button pure suitable anymore?

Reply:
New styles of `Button Pure` have a bigger hover padding which would be to big for the `Popover`. Also focus styles are also square with only rounded corner. Popover needs fully rounded focus.